Scripts/RemediationScripts/Readme.md

```diff
@@ -1016,6 +1019,48 @@ Yes
 
 ___ 
 
+
+
+## 49. Azure_AVD_Audit_Enable_HostPool_BootDiagnostics
+
+### Display Name
+Boot Diagnostic must be enabled with Managed Storage Account on Azure AVD Host pool VMs.
+
+### Link to Bulk Remediation Script (BRS)
+[Remediate-BootDiagnosticsForAVDHotPool](Remediate-BootDiagnosticsForAVDHotPool.ps1)
+
+### Minimum permissions required to run the script
+Contributor role at resource level
+
+### [Supports managed identity](Readme.md#supports-managed-identity-based-remediations) based remediation
+Yes
+
+### Supports rollback?
+Yes
+
+___ 
+
+
+
+## 50. Azure_AVD_SI_Configure_HostPool_SecureBoot
+
+### Display Name
+Azure AVD Host pool VMs should be of security type Trusted launch with Secure boot and vTPM enabled.
+
+### Link to Bulk Remediation Script (BRS)
+[Remediate-TrustedLaunchForAVDHotPool](Remediate-TrustedLaunchForAVDHotPool.ps1)
```

Comment:
This link doesn't work in UI - gives 404

Reply:
Updated the link now working.